com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -160,24 +290,70 @@ public bool IsCameraProjectionMatrixFlipped()
             return true;
         }
 
+        /// <summary>
+        /// The sorting criteria used when drawing opaque objects by the internal URP render passes.
+        /// When a GPU supports hidden surface removal, URP will rely on that information to avoid sorting opaque objects front to back and
+        /// benefit for more optimal static batching.
+        /// </summary>
         public SortingCriteria defaultOpaqueSortFlags;
 
         internal XRPass xr;
 
         [Obsolete("Please use xr.enabled instead.")]
         public bool isStereoEnabled;
 
+        /// <summary>
+        /// Maximum shadow distance visible to the camera. When set to zero shadows will be disable for that camera.
+        /// </summary>
         public float maxShadowDistance;
+
+        /// <summary>
+        /// True if post-processing is enabled for this camera.
+        /// </summary>
         public bool postProcessEnabled;
 
+        /// <summary>
+        /// Provides set actions to the renderer to be triggered at the end of the render loop for camera capture.
+        /// </summary>
         public IEnumerator<Action<RenderTargetIdentifier, CommandBuffer>> captureActions;
 
+        /// <summary>
+        /// The camera volume layer mask.
+        /// </summary>
         public LayerMask volumeLayerMask;
+
+        /// <summary>
+        /// The camera volume trigger.
+        /// </summary>
         public Transform volumeTrigger;
 
+        /// <summary>
+        /// If set to true, the integrated post-processing stack will kill NaNs generated by render passes prior to post-processing.
```

Comment:
This might be confusing to some new devs. Want to add that the value will be replaced with the color black/zero to make it a bit more explicit?

Reply:
fixed